scripts/config_comms_templates.py

```diff
@@ -1,28 +1,34 @@
+import argparse
 import requests
-from flask import json, current_app
+from flask import json
 
 headers = {'Content-type': 'application/json'}
 
+parser = argparse.ArgumentParser(description='Pass in a service URL')
+default_url = parser.add_argument('default_url', help='Main URL')
+arg_user = parser.add_argument('user', help='User')
+arg_password = parser.add_argument('password', help='Pass')
+args = parser.parse_args()
+
 # Script which adds new templates to the comms-service db along with the associated classifiers
 # Run with the cmd pipenv run python scripts/config_comms_templates.py
 with open('scripts/templates_to_load.json') as read_file:
     contents = json.load(read_file)
```

Comment:
You only need this line inside the `with` context manager. The rest doesn't need to be indented